Sources/SourceKitBazelBSP/SharedUtils/Shell/CommandRunner.swift

```diff
@@ -20,18 +20,27 @@
 import Foundation
 
 protocol CommandRunner {
-    func run(_ cmd: String, cwd: String?) throws -> String
+    func run(_ cmd: String, cwd: String?) throws -> Data
+}
+
+extension Data {
+    func toStr() -> String {
+        let str = String(data: self, encoding: .utf8) ?? ""
+        return str.trimmingCharacters(in: .whitespacesAndNewlines)
+    }
 }
```

Comment:
```suggestion
extension CommandRunner {
    func run(_ cmd: String, cwd: String?) throws -> String {
        let data: Data = try run(cmd, cwd: cwd)
        let str = String(data: data, encoding: .utf8) ?? ""
        return str.trimmingCharacters(in: .whitespacesAndNewlines)
    }

    func run(_ cmd: String) throws -> String {
        try run(cmd, cwd: nil)
    }
}
```

Since this is also trimming lines, I think it will be more accurate to keep the call under CommandRunner itself like it was before. You can then disambiguate by adding the type directly to the callers (`: String`).